docs/tutorial/kymotracking.rst

```diff
@@ -0,0 +1,206 @@
+
+The result of tracking is a list of kymograph traces::
+
+    >>> print(len(traces))  # the number of traces found in the kymo
+    745
+
+Sometimes, we can have very short spurious traces. To remove these from the list of detected traces we can use
+filter_lines. To omit all traces with fewer than 4 detected points, we can invoke::
+
+    >>> traces = lk.filter_lines(traces, 3)
+
+We can get the determined time and coordinate indices of these traces from the `time_idx` and `coordinate_idx`
+attributes. Let's plot the first trace::
+
+    plt.plot(traces[0].time_idx, traces[0].coordinate_idx)
+
+Plotting all the detected traces is also quite easy by iterating over the list of traces::
+
+    plt.imshow(data, aspect="auto", vmax=5)
+    for trace in traces:
+        plt.plot(trace.time_idx, trace.coordinate_idx)
+
+.. image:: kymotracked.png
+
+Fortunately, the signal to noise level in this kymograph is quite good. In practice, when the signal to noise is lower,
+one will have to resort to some fine tuning of the algorithm parameters over different regions of the kymograph to get
+an acceptable result.
+
+Note that the determined time and coordinates of the traces and coordinates are defined in pixels and therefore need to
+be converted to actual time and position using the delta time and pixel size of the kymograph. We can get the time
+between frames directly from the kymograph::
+
+    dt = kymo.delta_t
+
+And the pixel size as well::
+
+    pixel_to_um = kymo.pixelsize_um
+
+This allows conversion from pixels back to position.
+
+The second algorithm present is an algorithm that works purely on signal derivative information. It works by blurring
+the image, and then performing sub-pixel accurate line detection. It can be a bit more robust to low signal levels,
+but is generally less temporally and spatially accurate due to the blurring involved::
+
+    traces = lk.track_lines(data, line_width=3, max_lines=50)
+
+The interface is mostly the same, aside from an extra required parameter named `max_lines` which indicates the maximum
+number of lines we want to detect.
+
+
+Extracting summed intensities
+-----------------------------
+
+Sometimes, it can be desirable to extract pixel intensities in a region around our kymograph trace. We can quite easily
+extract these using the method `sample_from_image`. For instance, if we want to sum the pixels in a 9 pixel area around
+the longest kymograph trace, we can invoke::
+
+    plt.figure()
+    longest_trace_idx = np.argmax([len(trace) for trace in traces])
+    longest_trace = traces[longest_trace_idx]
+    plt.plot(np.array(longest_trace.time_idx) * kymo.delta_t, longest_trace.sample_from_image(num_pixels=5))
+    plt.xlabel('Time [s]')
+    plt.ylabel('Summed signal')
+
+Here `num_pixels` is the number of pixels to sum on either side of the trace. Note that data has to be the same image
+that the trace was based on.
+
+.. image:: kymo_sumcounts.png
+
+
+Algorithms
+----------
+
```

Comment:
This could be a bit more descriptive. Something along the lines of:
```suggestion
How the algorithms work 
-----------------------

```

Reply:
👍